tsconfig.json

```diff
@@ -7,11 +7,16 @@
     "lib": [
       "es2015"
     ],
-    "outDir": "build"
+    "outDir": "build",
+    "types": [
+      "node",
+      "node-fetch"
+    ]
```

Comment:
This restricts the default imported types so that we don't automatically get browser types (these literally mean look in the `node_modules/@types/` dir for type imports by the same name). Previously we imported all known types (which was a problem).